importer/helpers/dagbuilder.go

```diff
@@ -3,8 +3,13 @@ package helpers
 import (
 	dag "github.com/ipfs/go-ipfs/merkledag"
 	"github.com/ipfs/go-ipfs/pin"
+	u "github.com/ipfs/go-ipfs/util"
 )
 
+type BlockCB func(u.Key, bool) error
```

Comment:
> Oh weird, i could have sworn i tried it and it got angry...

It's all or nothing. If you name one, it yells at you until you name them all or `_`.